src/pages/FormPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
As you do not have any styled components in this file, I suggest to delete this import.

```suggestion
```